process/block/baseProcess.go

```diff
@@ -1239,3 +1242,66 @@ func (bp *baseProcessor) addHeaderIntoTrackerPool(nonce uint64, shardID uint32)
 		bp.blockTracker.AddTrackedHeader(headers[i], hashes[i])
 	}
 }
+
+func (bp *baseProcessor) commitTrieEpochRootHashIfNeeded(metaBlock *block.MetaBlock) error {
+	trieEpochRootHashStorageUnit := bp.store.GetStorer(dataRetriever.TrieEpochRootHashUnit)
+	if check.IfNil(trieEpochRootHashStorageUnit) {
+		return nil
+	}
+	_, isStorerDisabled := trieEpochRootHashStorageUnit.(*storageUnit.NilStorer)
+	if isStorerDisabled {
+		return nil
+	}
+
+	userAccountsDb := bp.accountsDB[state.UserAccountsState]
+	if userAccountsDb == nil {
+		return nil
+	}
+
+	currentRootHash, err := userAccountsDb.RootHash()
+	if err != nil {
+		return err
+	}
+	epochBytes := make([]byte, 4)
+	binary.BigEndian.PutUint32(epochBytes, metaBlock.Epoch)
```

Comment:
please use bp.uint64Converter instead of big endian (to be consistent with the rest of the project)

Reply:
done